### ISSUE TITLE
Confused with "e fi te, lo kin kekeko fi ka"

### ISSUE THREAD

**Issue by brunobord** [mellow-mellow/si-ka#2]
refs: http://www.reddit.com/r/conlangs/comments/34ixo5/just_used_5_minutes_of_your_day_day_296/cqvd41r

I've compared this sentence with the [vocabulary available on this github repository](https://github.com/mellow-mellow/si-ka/blob/master/words.csv), and was a bit confused:

```
le fi te, lo kin kekeko fi ka.
```

The words.csv file says:

* `le` means "sound", rather than "light", the word light is `lia`
* `fi` means "intent", while the word "effect" is `fe`
* `kin` does not exist... is it the word `ki` with a final "n" to ease the pronunciation?
* `kekeko` : I can't translate this as "001". There's no explanation on how numbers are made.

So here's my question: the `words.csv` file doesn't seem very old, although it presents differences looking at this sentence. So which one is the "true" si-ka?

Anyway, I quite like you minlang, it looks like it could be usable at some point.

**Comment by mellow-mellow** [comment]
Thank you! It's nice to see interest in my language. The current vocabulary is out of date; it's be more coherent when I get back to this project, since I've been working on other things. In brief, I've made (undocumented) changes to the vocabulary since my last push, and the more recent usage (the one on reddit) is accurate. That is,

* `le` more accurately means a signal that propagates through a physical medium, and can refer to light or sound. This is because the distinction is rarely important, and when it is, one can distinguish between "direction `le`", which would be directed like light, and "topological `le`", which moves through a space like sound. (I don't have words for all these yet.)
* I decomposed the old notion of intent into "effect immediate-vicinity" or something like that, so `fi` gets the meaning "effect" now. To understand my translation of intent for si-ka, consider this scenario: Someone pushes a small boat across a pond, hoping that it will reach the other side. If nothing else happens, it does, but if the waves interfere, the boat might take a different trajectory. In that sense, the push of the boat encapsulates the intent, assuming the thought process of the person is straightforward. This gets more nuanced in cases of accidents or mental illness, where the causality of a single person needs to be broken down further, but that's the overview.
* I needed a word for "code" or "representation", so I coined one based on some sound ideas I had, like usual. That was probably the first public usage of `kin`.
* `kekeko` is a single word (and so is phonologically different from `ke ke ko` as per si-ka phonotactics), and it breaks down as "none-none-unit", or 0 0 1. I don't have words for many higher numbers yet, nor does si-ka use base 10, but since a change of base is irrelevant in that context ("a hundred" just has to sound big), I went with a relatively close option. I plan to have a less ad-hoc number system soon.

Si-ka honestly isn't very well defined in its current state, but I'll be back with a great deal of new documentation soon to clarify numerous aspects. It's a bit of a slow process, but it'll be worth it.
